herd/ASLSem.ml

```diff
@@ -394,6 +394,12 @@ module Make (Conf : Config) = struct
           return v
       | _ -> M.op (Op.ArchOp ASLOp.Concat) v1 v2
 
+    let binop_mod v1 v2 =
+      match (v1, v2) with
+      | (V.Val Constant.(Symbolic _| Label _) | V.Var _), V.Val (Constant.Concrete (ASLScalar.S_Int _)) ->
+          return V.zero
```

Comment:
This seems slightly unusual

Reply:
Indeed. I've reduced this behaviour to only check for the least significant bits (i.e. to bypass the `IsAligned` checks).